coreneuron/network/netcvode.cpp

```diff
@@ -576,26 +576,12 @@ void SelfEvent::pr(const char* s, double tt, NetCvode*) {
 }
 
 void ncs2nrn_integrate(double tstop) {
-    double ts;
-    int n = (int)((tstop - nrn_threads->_t) / dt + 1e-9);
+    int total_sim_steps = (int)((tstop - nrn_threads->_t) / dt + 1e-9);
```

Comment:
sorry for nitpicking, but could you please use C++-style casts instead? as you are changing this line already

Reply:
Good point :+1: This was an oversight